sem_covid/entrypoints/etl_dags/eurlex_covid19.py

```diff
@@ -62,192 +62,351 @@
-        }
-        OPTIONAL {
-            ?work cdm:work_date_creation ?date_created .
-        }
-        OPTIONAL {
-            ?work cdm:resource_legal_date_signature ?legal_date_signature .
-        }
-        OPTIONAL {
-            ?work cdm:resource_legal_date_entry-into-force ?legal_date_entry_into_force .
-        }
-    
-        # metadata - identification properties
-        OPTIONAL {
-            ?work cdm:resource_legal_id_celex ?celex .
-        }
-        OPTIONAL {
-            ?work cdm:resource_legal_eli ?legal_eli .
-        }
-        OPTIONAL {
-            ?work cdm:work_id_document ?id_document .
-        }
-        OPTIONAL {
-            ?work owl:sameAs ?same_as_uri .
-        }
-    
-        # diving down FRBR structure for the title and the content
-        OPTIONAL {
-            ?exp cdm:expression_belongs_to_work ?work ;
-                 cdm:expression_uses_language ?expr_lang ;
-                 cdm:expression_title ?title .
-    
             OPTIONAL {
-                ?manif_pdf cdm:manifestation_manifests_expression ?exp ;
-                           cdm:manifestation_type ?type_pdf .
-                FILTER (str(?type_pdf) in ('pdf', 'pdfa1a', 'pdfa2a', 'pdfa1b', 'pdfx'))
+                ?work owl:sameAs ?same_as_uri .
             }
+
             OPTIONAL {
-                ?manif_html cdm:manifestation_manifests_expression ?exp ;
-                            cdm:manifestation_type ?type_html .
-                FILTER (str(?type_html) in ('html', 'xhtml'))
+                ?exp cdm:expression_belongs_to_work ?work ;
+                     cdm:expression_uses_language ?expr_lang ;
+                     cdm:expression_title ?title .
+
+                OPTIONAL {
+                    ?manif_pdf cdm:manifestation_manifests_expression ?exp ;
+                               cdm:manifestation_type ?type_pdf .
+                    FILTER (str(?type_pdf) in ('pdf', 'pdfa1a', 'pdfa2a', 'pdfa1b', 'pdfx'))
+                }
+                OPTIONAL {
+                    ?manif_html cdm:manifestation_manifests_expression ?exp ;
+                                cdm:manifestation_type ?type_html .
+                    FILTER (str(?type_html) in ('html', 'xhtml'))
+                }
             }
+            BIND(IRI(concat(?manif_pdf,"/zip")) as ?pdf_to_download)
+            BIND(IRI(concat(?manif_html,"/zip")) as ?html_to_download)
         }
-        BIND(IRI(concat(?manif_pdf,"/zip")) as ?pdf_to_download)
-        BIND(IRI(concat(?manif_html,"/zip")) as ?html_to_download)
-    }
-    GROUP BY ?work ?title
-    ORDER BY ?work ?title"""
+        GROUP BY ?work ?title
+        ORDER BY ?work ?title"""
+
+sources = {
+    "EurLex": {"json": config.EURLEX_TIMELINE_JSON, "query": eurlex_query},
+    "Extended EurLex": {"json": config.EURLEX_EXTENDED_JSON, "query": eurlex_extended_query}
+}
+
+
+def get_transformation_rules(rules: str, search_rule: str = SEARCH_RULE):
```

Comment:
this is no longer needed, is it?

Reply:
it is, it's the eurlex dag (now we have extracted just for the pwdb dag)